docs/application/web/guides/data/file-system.md

```diff
@@ -1,13 +1,13 @@
 # File System
 
-> **Note**  
+> [!NOTE]  
 > 
 > With the introduction of Tizen version 5.0, the Filesystem API has undergone a major overhaul. Many of the existing methods are deprecated and new methods are introduced. For information on the previous versions of API, see [Deprecated Filesystem Guide](./file-system-old.md). This guide explains about the APIs for Tizen 5.0 and its newly introduced methods.
```

Comment:
```suggestion
> [!NOTE]  
> With the introduction of Tizen version 5.0, the Filesystem API has undergone a major overhaul. Many of the existing methods are deprecated and new methods are introduced. For information on the previous versions of API, see [Deprecated Filesystem Guide](./file-system-old.md). This guide explains about the APIs for Tizen 5.0 and its newly introduced methods.
```

Global.

It is confusable, but the new style note delete the middle empty line.

https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning

Reply:
@mijin85cho, fixed all the note related issues.